pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+
+For attaching clusters _with_ networking restrictions, the Kubernetes version of the cluster you wish to attach must be greater than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
+
+When working with clusters that have networking restrictions, you need to run some YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel.
+
+### Projects and Workspaces
+
+Before you attach clusters, you need to create one or more Projects and Workspaces. [Projects][projects] allow you to define one or more clusters as a group to which Kommander pushes a common configuration. [Workspaces][workspaces] give you a logical way to represent your teams and specific configurations. Grouping your existing clusters in Kommander projects and workspaces makes managing their plaform services and resources easier and supports monitoring and logging.
+
+### Platform service requirements
+
+In addition to the basic cluster requirements, the combination of platform services that Kommander will manage on those clusters is very important. The [specific set of platform services][platform_service_requirements] you want to use makes a big difference in the requirements for the cluster nodes and their resources (CPU, memory, and storage).
+
+The default platform services that Kommander provides include Prometheus, Kubecost, Traefik, and cert-manager.
+
+## Attaching existing AWS and EKS clusters
+
+Attaching an existing, Amazon-provisioned cluster requires that the cluster be fully-configured and running. The suggested default cluster confguration includes a control plane pool containing three (3) m5.xlarge nodes and a worker pool containing four (4) m5.2xlarge nodes.
+
+Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
+
+To attach an existing EKS cluster, refer to the specific information in [Attach Amazon EKS Cluster to Kommander][attach_eks_cluster].
+
+## Attaching existing Azure clusters
+
+Attaching an existing, Azure-provisioned cluster requires that the cluster be fully-configured and running. The suggested default cluster configuration includes a control plane pool containing three (3) Standard_DS3_v2 nodes and a worker pool containing six (6) Standard_DS3_v2 nodes.
+
+Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
+
+[attach_eks_cluster]: /dkp/kommander/latest/clusters/attach-cluster/attach-eks-cluster/
```

Comment:
I'm of two minds on /latest/ but I feel like we should version the docs, because if you're in the beta area, it'll push you down to the previous version (which may or may not have the same link description), or if you're at a lower version, it'll take you to the newest, which may not match the functionality that we now have, and it does seem like the different functionalities are diverging greater and greater. So, I'm going to suggest we actually version these, but I don't feel too strongly about it. Just a little bit: 

```suggestion
[attach_eks_cluster]: /dkp/kommander/1.4/clusters/attach-cluster/attach-eks-cluster/
```

Reply:
Changed as noted.